src/igvfd/upgrade/user.py

```diff
@@ -29,3 +29,9 @@ def user_3_4(value, system):
     if 'viewing_groups' in value:
         if len(value['viewing_groups']) < 1:
             del value['viewing_groups']
+
+
```

Comment:
i think the upgrade should not be empty but should be ready to handle the possibility of erroneous email

Reply:
the upgrade should validate against the regex "pattern": "^[^\\s@]+@[^\\s@]+\\.[^\\s@]+$"